README.md

```diff
@@ -299,7 +300,7 @@ metadata:
 # ...
 ```
 
-### SSL
+### <a name="ssl"></a>SSL
```

Comment:
That's not an oops.  That's a legitimate way to insert an anchor that can be referenced by other hyperlinks in the same document.  If there's a better way to do that, I'm not sure what it is.